projects/com.nvidia.grcuda/src/com/nvidia/grcuda/gpu/CUDARuntime.java

```diff
@@ -270,6 +270,12 @@ public String cudaGetErrorString(int errorCode) {
      */
     @TruffleBoundary
     public Object getSymbol(String libraryPath, String symbolName, String signature) throws UnknownIdentifierException {
+        return getSymbol(libraryPath, symbolName, signature);
```

Comment:
BUG: This gives an infinite recursion.

Reply:
unittests fail.

Reply:
oh, wow, I introduced that while dissecting the changes into separate PRs.